### PR TITLE
docs(serializers): Fix grammatical errors

### PR DESCRIPTION
**Pull request by jamhour1g**

### CODE REVIEW

**Comment by sandwwraith** [comment]
Hi, thanks for all your corrections. I see that you've opened a lot of pull requests with small changes. May I ask you to combine all of them into one PR? It would be easier to review and merge it that way.

**Comment by jamhour1g** [comment]
@sandwwraith, thanks for the input. I'll merge the pull requests as suggested.

Just to clarify, the reason for the initial split into several small PRs was to make sure each update could be reviewed on its own, to simplify the management and discussion of each focused change, to reduce the chance of merge conflicts with other ongoing updates, and to make it easier to reject any non-essential changes by keeping them separate.

I'm on it—starting the merge into one big PR now. If you need more details or have a preferred structure for the combined PR, just let me know. Should I go ahead and close the previous PRs?

**Comment by jamhour1g** [comment]
@sandwwraith, should I commit the API dump files created using `./gradlew apiDump`? There were changes to the `encodeSerializableValue` and `decodeSerializableValue` docs.

**Comment by sandwwraith** [comment]
@jamhour1g API dump uses only signatures of the functions, so there is no need in that. `./gradlew apiCheck` also passes fine.

File: docs/serializers.md [hunk 1]
```diff
@@ -851,7 +864,7 @@ fun main() {
 
 <!--- TEST --> 
 
-### Specifying serializer globally using typealias
+### Specifying a serializer globally using a typealias
```
[the remark this answers is not in the document]
Comment (jamhour1g) [reply docs/serializers.md:867]: @sandwwraith, thanks for the heads-up. It's taken care of. I've also inserted the article "a" in the adjacent sections.

**Review by sandwwraith — APPROVED** [review]
Thank you for all your efforts!

**Pull request merged**